infra/base-images/base-builder/compile_python_fuzzer

```diff
@@ -74,6 +77,28 @@ EOF
   mv tmp_fuzzer_coverage.py $fuzzer_path
 fi
 
+# If PYSECSAN is enabled, ensure that we can build with it.
+if [[ ${ENABLE_PYSECSAN:-"NO"} != "NO" ]];
```

Comment:
Would probably use 0 for this. NO is pretty idiosyncratic.

Reply:
Thanks! Done!